tests/unit/test_app_profile.py

```diff
@@ -510,49 +531,59 @@ def test_update_app_profile_routing_any(self):
         # Create response_pb
         metadata = messages_v2_pb2.UpdateAppProfileMetadata()
         type_url = "type.googleapis.com/{}".format(
-            messages_v2_pb2.UpdateAppProfileMetadata.DESCRIPTOR.full_name
+            messages_v2_pb2.UpdateAppProfileMetadata._meta._pb.DESCRIPTOR.full_name
         )
         response_pb = operations_pb2.Operation(
             name=self.OP_NAME,
-            metadata=Any(type_url=type_url, value=metadata.SerializeToString()),
+            metadata=Any(type_url=type_url, value=metadata._pb.SerializeToString()),
         )
 
         # Patch the stub used by the API method.
-        channel = ChannelStub(responses=[response_pb])
-        instance_api = bigtable_instance_admin_client.BigtableInstanceAdminClient(
-            channel=channel
-        )
+        instance_api = mock.create_autospec(BigtableInstanceAdminClient)
         # Mock api calls
+        instance_api.app_profile_path.return_value = (
+            "projects/project/instances/instance-id/appProfiles/app-profile-id"
+        )
+
         client._instance_admin_client = instance_api
 
         # Perform the method and check the result.
         ignore_warnings = True
         expected_request_update_mask = field_mask_pb2.FieldMask(
             paths=["description", "single_cluster_routing"]
         )
-        expected_request = messages_v2_pb2.UpdateAppProfileRequest(
-            app_profile=app_profile._to_pb(),
-            update_mask=expected_request_update_mask,
-            ignore_warnings=ignore_warnings,
-        )
 
+        expected_request = {
+            "request": {
+                "app_profile": app_profile._to_pb(),
+                "update_mask": expected_request_update_mask,
+                "ignore_warnings": ignore_warnings,
+            }
+        }
+
+        instance_api.update_app_profile.return_value = response_pb
+        app_profile._instance._client._instance_admin_client = instance_api
         result = app_profile.update(ignore_warnings=ignore_warnings)
-        actual_request = channel.requests[0][1]
+        actual_request = client._instance_admin_client.update_app_profile.call_args_list[
+            0
+        ].kwargs
 
         self.assertEqual(actual_request, expected_request)
-        self.assertIsInstance(result, operation.Operation)
-        self.assertEqual(result.operation.name, self.OP_NAME)
-        self.assertIsInstance(result.metadata, messages_v2_pb2.UpdateAppProfileMetadata)
+        # self.assertIsInstance(result, operation.Operation)
```

Comment:
@crwilcox I have a few instances like this where after we refactored this test (from using channels), we are no longer getting an `operation.Operation` back, but a pb2 Operation. Are we missing anything by removing these assertions?